py/obj.c

```diff
@@ -67,10 +67,10 @@ const mp_obj_type_t *MICROPY_WRAP_MP_OBJ_GET_TYPE(mp_obj_get_type)(mp_const_obj_
     } else if (mp_obj_is_obj(o_in)) {
         const mp_obj_base_t *o = MP_OBJ_TO_PTR(o_in);
         return o->type;
-    #if MICROPY_PY_BUILTINS_FLOAT
+        #if MICROPY_PY_BUILTINS_FLOAT
```

Comment:
is it possible to not reformat these #'s ?

Reply:
Ah whoops, the first ~5 times I pushed this branch I undid all the auto-codeformat on these lines, must have missed that on the current / last push. Not sure why my uncrustify wants to do this here :-(